/usr/bin/Xwrapper: symbol lookup error: /usr/lib/xorg/modules/drivers//vesa_drv.so: undefined symbol: xf86GTFMode

ref: http://bugs.debian.org/489051

--- xf86-video-vesa-2.0.0/src/vesa.c~	2008-07-01 19:40:04.000000000 +0200
+++ xf86-video-vesa-2.0.0/src/vesa.c	2008-08-21 21:30:41.964937080 +0200
@@ -342,7 +342,12 @@
      * then, by this point, you're well into despair territory.
      */
     for (v = mon->vrefresh[0].lo; v <= mon->vrefresh[0].hi; v++) {
+#if XORG_VERSION_CURRENT > XORG_VERSION_NUMERIC(1,4,2,0,0)
+        /* xf86GTFMode is only avalaible in newer versions of Xorg */
 	mode = xf86GTFMode(p->HDisplay, p->VDisplay, v, 0, 0);
+#else
+	mode = xf86CVTMode(p->HDisplay, p->VDisplay, v, FALSE, FALSE);
+#endif
 	ret = xf86CheckModeForMonitor(mode, mon);
 	xfree(mode);
 	if (ret == MODE_OK)
